### PR TITLE
feat: add back to liveApp analytics

### PR DESCRIPTION
**Pull request by brunospolidorio**
### 📝 Description

Add analytics when clicking on Back to LiveApp

### ❓ Context

- **Impacted projects**: `LLD and LLM` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-5324` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by ggilchrist-ledger** [comment]
Working on LLD 
image
image

File: apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx [hunk 1]
```diff
@@ -45,13 +52,37 @@ function BackToWhitelistedDomain({ manifest }: BackToWhitelistedDomainProps) {
       >
     >();
 
+  const getButtonLabel = () => {
+    if (manifest.id === "multibuy") {
```
[the remark this answers is not in the document]
Comment (brunospolidorio) [reply apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx:56]: If manifest.id === null we don't have a LiveApp to be rendered. We never gets to this part if manifest is broken

**Comment by ggilchrist-ledger** [comment]
Working now on Mobile 
image

File: apps/ledger-live-desktop/src/renderer/components/WebPTXPlayer/TopBar.tsx [hunk 2]
```diff
@@ -123,12 +124,33 @@ export const TopBar = ({ manifest, webviewAPIRef, webviewState }: Props) => {
...
+      flow: flowName,
+    });
+
+    await webview.loadURL(safeUrl);
     webview.clearHistory();
-  }, [webviewAPIRef]);
+  }, [webviewAPIRef, webviewState.url]);
+
+  const getButtonLabel = useCallback(() => {
+    if (manifest.id === "multibuy") {
```
Comment (gre) [inline apps/ledger-live-desktop/src/renderer/components/WebPTXPlayer/TopBar.tsx:145]: it feels wrong to `if ( specific live app )` in the generic codebase 🤔

**Review by ggilchrist-ledger — APPROVED** [review]
All good from my point of view

**Pull request merged**